components/templates/Article.tsx

```diff
@@ -74,11 +75,15 @@ const Article: React.FunctionComponent<Props> = props => {
 
             {/* Title and author */}
             <h1>{props.title}</h1>
-            {props.author && <p className="text-muted">By {props.author}</p>}
+            { props.authorId &&
+                <Link href={`/authors/${props.authorId}`}>
+                    <a className="text-muted text-capitalize mb-5">By {props.author}</a>
```

Comment:
Can we style this to remove `text-decoration`?

Reply:
Yes Katy, we can, but how do we make this an obvious link, without changing the color of the text?

Reply:
I would add `cursor: pointer;` to the custom style.